CONTRIBUTING.md

```diff
@@ -1,62 +1,73 @@
 # Contributing to void-packages
 
-void-packages is the backbone of the Void Linux distribution. It contains all definitions how packages are built from source.
+void-packages is the backbone of the Void Linux distribution. It contains all the necessary definitions to build packages from source.
 
-This document describes how you as a contributor can help adding packages, correcting bugs and adding features to void-packages.
+This document describes how you, as a contributor, can help with adding packages, correcting bugs and adding features to void-packages.
 
 ## Getting your packages into Void by yourself
 
-If you really want to get a package into Void Linux we recommend you to package it yourself.
+If you really want to get a package into Void Linux, we recommend you package it yourself.
+
 We provide a [comprehensive Manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
-on how you can create new packages. Also there's a
-[manual about xbps-src](https://github.com/void-linux/void-packages/blob/master/README.md) which is used
+on how to create new packages. There's also a
+[manual for xbps-src](https://github.com/void-linux/void-packages/blob/master/README.md), which is used
 to build package files from templates.
 
-For this guide, we assume you have basic knowledge about [git](http://git-scm.org) and a [GitHub Account](http://github.com)
+For this guide, we assume you have basic knowledge about [git](http://git-scm.org), as well as a [GitHub Account](http://github.com).
 
-Please note that we do not accept any packages containing non-release versions such
-as specific git- or svn-revisions anymore.
```

Comment:
This "anymore" is significant, there are still previously added snapshots in repo.

Reply:
Ah okay, thanks, i'll re-add that.